src/views/virtualmachines/list/hooks/useVirtualMachineColumns.ts

```diff
@@ -66,7 +68,7 @@ const useVirtualMachineColumns = (namespace: string) => {
         props: { className: 'pf-m-width-10' },
       },
       {
-        title: '',
+        title: ' ',
```

Comment:
I don't understand the reason of this change. IMO it is unnecessary.

Reply:
reverted